composer.json

```
@@ -24,7 +24,7 @@
         "yiisoft/friendly-exception": "^1.0",
         "yiisoft/html": "^2.0",
         "yiisoft/strings": "^2.0",
-        "yiisoft/view": "^4.0"
+        "yiisoft/view": "^5.0"
```

```suggestion
        "yiisoft/view": "^4.0|^5.0"
```

We can leave 4.0 version and make patch release instead of major.

---

I think it's better not to leave version 4, because the public API is changed in yiisoft/views, and if you leave version 4, it can be confusing.

---

No, it's fine since this changed public API isn't used in this package.